web/packages/plugins/ui/analysis/src/ExpertView.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react'
+import { DmtUIPlugin, UIPluginSelector } from '@dmt/common'
+import { AnalysisInfoCard, AnalysisJobTable } from '@dmt/analysis-platform'
```

Comment:
These components, that are only used by these plugins, should be in this plugin package.
There should not be dependencies from plugins to any other "custom package" that "dmt/common"